projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -730,6 +731,16 @@ function wpcom_launchpad_get_task_definitions() {
 	return array_merge( $extended_task_definitions, $task_definitions );
 }
 
+/**
+ * Returns true if the current site is launched.
+ *
+ * @return boolean
+ */
+function wpcom_launchpad_is_site_launched() {
```

Comment:
We should also check if the status is already complete, so we don't need always to check the option:
https://github.com/Automattic/jetpack/blob/ff17b59347cd29f9113174420ced10c7526670f4/projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php#L1058-L1061